WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/OrderCustomerSection.swift

```diff
@@ -39,7 +39,7 @@ struct OrderCustomerSection: View {
                     }
             }
         }
-        .sheet(isPresented: $viewModel.showsAddressForm) {
+        .fullScreenCover(isPresented: $viewModel.showsAddressForm) {
```

Comment:
@staskus  This is the one for which you wrote

> Since this view displays more than a dozen fields, I think it makes sense to explicitly display it in full screen [to make all fields visible].

If that's the case, sounds good to me.